### PR TITLE
feat(console): allow collapsing and expanding map nodes

### PR DESCRIPTION
**Pull request by polamoros**
Resolves https://github.com/winglang/wing/issues/6624

### CODE REVIEW

File: apps/wing-console/console/ui/src/features/map-view.tsx [hunk 1]
```diff
@@ -37,25 +34,38 @@ const PORT_ANCHOR = 0;
 const EDGE_ROUNDED_RADIUS = 10;
 // For more configuration options, refer to: https://eclipse.dev/elk/reference/options.html
 const baseLayoutOptions: LayoutOptions = {
+  "elk.interactiveLayout": "true",
```
Comment (skyrpex) [inline apps/wing-console/console/ui/src/features/map-view.tsx:37]: So is this actually working?

File: apps/wing-console/console/ui/src/features/map-view.tsx [hunk 2]
```diff
@@ -98,9 +108,33 @@ const Wrapper: FunctionComponent<PropsWithChildren<WrapperProps>> = memo(
...
+                    "hover:text-sky-600 dark:hover:text-sky-300 transition-colors",
+                  )}
+                />
+              )}
+            </div>
+          </div>
         </div>
-
-        {children}
+        {collapsed || children}
```
Comment (skyrpex) [inline apps/wing-console/console/ui/src/features/map-view.tsx:137]: Won't this render something like a `1` or `true` if collapsed? 🤔

**Comment by ainvoner** [comment]
@polamoros can you also fix this check annotation issue: apps/wing-console/console/ui/src/ui/resource-metadata.tsx (see `files changed`)

File: apps/wing-console/console/ui/src/features/map-view.tsx (hunk 2 above)
Comment (polamoros) [reply apps/wing-console/console/ui/src/features/map-view.tsx]: React is not rendering booleans 
image

File: apps/wing-console/console/ui/src/features/map-view.tsx (hunk 1 above)
Comment (polamoros) [reply apps/wing-console/console/ui/src/features/map-view.tsx:37]: not really.. removing it for now

File: apps/wing-console/console/ui/src/features/map-view.tsx (hunk 2 above)
Comment (skyrpex) [reply apps/wing-console/console/ui/src/features/map-view.tsx]: Perfect

**Review by skyrpex — APPROVED** [review]

**Pull request merged**